addons/website_cf_turnstile/static/src/interactions/turnstile.js

```diff
@@ -26,7 +26,7 @@ export class TurnStile {
         };
         // `this` is bound to the turnstile widget calling the callback
         globalThis.turnstileSuccess = function () {
-            const form = this.wrapper.closest("form");
+            const form = this.wrapper.closest("form") || this.wrapper.parentElement.parentElement;
```

Comment:
Is this one actually necessary?

Reply:
This seems sub-optimal but I can't really think of a nice way to do this either. I guess we could store a selector as a data attribute on the container div, and then use that to get the "form"... but that seems way over-engineered so let's stick with this.

Main issue is that in theory you could have the submit button in a separate div from the widget container (for whatever design reason). Let's cross that bridge if it ever comes up.